### PR TITLE
wcag 2.1: input styles

### PR DESCRIPTION
**Pull request by margaree**
also cleaned up old search scss

### CODE REVIEW

File: sass/deprecated/input-helpers.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		$background-color: $d2l-color-white,
 		$background-color-hover: null,
 		$border-radius: 0.3rem,
-		$box-shadow: inset 0 2px 0 0 rgba(185, 194, 208, .2),
+		$box-shadow: inset 0 2px 0 0 rgba(181, 189, 194, .2),
```
Comment (dlockhart) [inline sass/deprecated/input-helpers.scss]: Could you throw a comment here `/* corundum */` just so we know in the future that this is what we expected?

File: sass/deprecated/input-helpers.scss [hunk 2]
```diff
@@ -122,8 +122,7 @@
 	) {
 
 	@include _vui-input(
-		$background-color: $d2l-color-woolonardo,
-		$background-color-hover: $d2l-color-gypsum,
```
Comment (dlockhart) [inline sass/deprecated/input-helpers.scss]: Hmm, so I guess this was wrong before?

File: sass/deprecated/inputs.scss [hunk 3]
```diff
@@ -33,25 +32,6 @@ textarea.vui-input {
...
-.vui-input-search-container {
-	@include _vui-input-search-container();
-}
-.vui-input-search-button {
-	@include _vui-input-search-button();
-}
-.vui-input-search-clear-button {
-	@include _vui-input-search-clear-button();
-}
-
```
Comment (dlockhart) [inline sass/deprecated/inputs.scss]: Thanks again for finding these!

File: sass/deprecated/input-helpers.scss (hunk 2 above)
Comment (margaree) [reply sass/deprecated/input-helpers.scss]: yup it was at least inconsistent with `d2l-input*` as well as the "before" design in the story

**Pull request merged**